src/emeus-constraint-layout.c

```diff
@@ -1908,9 +1917,8 @@ emeus_constraint_layout_child_get_preferred_size (EmeusConstraintLayoutChild *se
   int child_nat = 0;
   Variable *attr = NULL;
 
-  switch (orientation)
+  if (self->solver && orientation == GTK_ORIENTATION_HORIZONTAL)
```

Comment:
I wanted to set minimum_p and natural_p to 0 if there was no solver
but now that I think about it, it does not really matter